nixos/modules/services/backup/restic.nix

```diff
@@ -251,6 +251,7 @@ in
             RuntimeDirectory = "restic-backups-${name}";
             CacheDirectory = "restic-backups-${name}";
             CacheDirectoryMode = "0700";
+            PrivateTmp = "yes";
```

Comment:
Maybe use `PrivateTmp = true` here? When grepping through the tree, I see most modules use `true` rather then a string.

Reply:
Thanks for the suggestion - consistency is always preferred. For the sake of it, I looked up what's happening under the hood.

`PrivateTmp=` is not explicitly handled in [`systemd-unit-options.nix`](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/system/boot/systemd-unit-options.nix) and thus is just treated as generic unit option by [`toOption`](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/system/boot/systemd-lib.nix#L106) which doesn't really prefer the one or another.